source/connectors/kafka-sink-cdc.txt

```diff
@@ -0,0 +1,78 @@
+.kafka-sink-cdc
+
+==============================
+Kafka Sink Change Data Capture
+==============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecols
+
+Change Data Capture
+-------------------
+
+
+Change Data Capture Mode
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The sink connector can also be used in a different operation mode in order to handle change data capture (CDC) events.
+Currently, the following CDC events from `Debezium <http://debezium.io/>`_ can be processed:
+
+
+* `MongoDB <http://debezium.io/docs/connectors/mongodb/>`_
+* `MySQL <http://debezium.io/docs/connectors/mysql/>`_
+* `PostgreSQL <http://debezium.io/docs/connectors/postgresql/>`_
+
+This effectively allows to replicate all state changes within the source databases into MongoDB collections. Debezium produces very similar
+CDC events for MySQL and PostgreSQL. The so far addressed use cases worked fine based on the same code which is why there is only one
```

Comment:
code which -> code, which

I think this sentence could be reworked a bit though so as to be less wordy.